backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -950,16 +956,23 @@ bool SurfaceSdlGraphicsManager::loadGFXMode() {
 		}
 #endif
 
-#if SDL_VERSION_ATLEAST(2, 0, 0)
+#ifdef MIYOOMINI
+		Uint32 flags = SDL_HWSURFACE;
+#elif SDL_VERSION_ATLEAST(2, 0, 0)
 		Uint32 flags = SDL_SWSURFACE;
 #else
 		Uint32 flags = _videoMode.isHwPalette ? (SDL_HWSURFACE | SDL_HWPALETTE | SDL_DOUBLEBUF) : SDL_SWSURFACE;
 #endif
-#ifndef RS90
+#if !defined(RS90) && !defined(MIYOOMINI)
```

Comment:
This (and one other RS90 ifdef) could potentially be removed after changing `OSystem_SDL_Opendingux::initBackend()` to enforce the `fullscreen` option to false instead of true on platforms that need it.

Reply:
Moved